goathacks/templates/header.html

```diff
@@ -36,7 +36,7 @@
               <a class="nav-link" href="../static/assets/Hack@WPI-SponsorshipInformation-2020.pdf">Sponsorship Packet</a>
             </li> -->
             <li class="nav-item">
-              <a class="nav-link" href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">Code of Conduct</a>
+              <a class="nav-link" href="https://hack.wpi.edu/conduct/GoatHacks-Code-of-Conduct.pdf">Code of Conduct</a>
```

Comment:
```suggestion
              <a class="nav-link" href="{{ url_for('conduct') }}">Code of Conduct</a>
```

You always want to use `url_for` in case the root URL or the path to the route changes.